src/transport/SecureSessionMgr.h

```diff
@@ -36,17 +36,19 @@
 #include <transport/PeerConnections.h>
 #include <transport/SecurePairingSession.h>
 #include <transport/SecureSession.h>
+#include <transport/TransportMgr.h>
 #include <transport/raw/Base.h>
 #include <transport/raw/Tuple.h>
 
 namespace chip {
 
-class SecureSessionMgrBase;
+class SecureSessionMgr;
+class TransportMgrBase;
```

Comment:
Includes please. Includes identify circular references which can indicate architectural issues we need to fix.